mkclient.sh failed with tsc error

Steps to reproduce this error:

1. `./client/bin/mkclient.sh`
2. `cd kui-stage; npm run webpack`
3. See `kui-compile` failed with the following error

```error TS18003: No inputs were found in config file '/Users/mengting.yan1ibm.com/Documents/kui6/kui/kui-stage/tsconfig.json'. Specified 'include' paths were '[]' and 'exclude' paths were '["node_modules/@kui-shell/builder/**/flycheck*.ts"]'.```


This is a long-existing compilation error, but we were good with Typescript 3.6.4. However, we didn't pin the version of Typescript in `mkclient.sh`, so `kui-compile` failed with Typescript ^3.7.2.

--- a/clients/bin/mkclient.sh
+++ b/clients/bin/mkclient.sh
@@ -37,12 +37,6 @@ npm run -s pack
 
 cd "$tmpdir"
 
-cat > tsconfig.json <<EOF
-{
-  "extends": "./node_modules/@kui-shell/builder/tsconfig-base.json"
-}
-EOF
-
 mkdir plugins
 cp -a -L ../clients/default/theme .
 

--- a/packages/builder/bin/compile.sh
+++ b/packages/builder/bin/compile.sh
@@ -29,17 +29,17 @@ if [ ! -d plugins ] ; then
 fi
 
 if [ ! -f tsconfig.json ]; then
-    echo "Error: missing tsconfig.json"
+    echo "Warning: skipped compiling Typescript because of missing tsconfig.json"
+else
+  # make typescript happy, until we have the real prescan model ready
+  # (produced by builder/lib/configure.js, but which cannot be run
+  # until after we've compiled the source)
+  mkdir -p ./node_modules/@kui-shell
+  touch ./node_modules/@kui-shell/prescan.json
+
+  # compile the source
+  npx tsc -b .
 fi
-
-# make typescript happy, until we have the real prescan model ready
-# (produced by builder/lib/configure.js, but which cannot be run
-# until after we've compiled the source)
-mkdir -p ./node_modules/@kui-shell
-touch ./node_modules/@kui-shell/prescan.json
-
-# compile the source
-npx tsc -b .
 
 # initialize the html bits
 CLIENT_HOME="$CLIENT_HOME" KUI_STAGE="$CLIENT_HOME" node node_modules/@kui-shell/builder/lib/configure.js

